schema.graphql

```diff
@@ -69,8 +69,8 @@ type Location {
   trafficCams: [TrafficCam!]!
   weather: Weather!
   widgets: WidgetCollection!
-  wifiName: String!
-  wifiPassword: String!
+  wifiName: String
+  wifiPassword: String
```

Comment:
What does removing '!' do/mean?

Reply:
This is the graphql schema Interface Definition Language, IDL. `!` means a field is not null:
https://graphql.org/learn/schema/#lists-and-non-null

Reply:
caused by the `null: true` that got added over here https://github.com/mojotech/helios2/pull/237/files#r641690789